src/main/webapp/js/media_manager.js

```diff
@@ -980,7 +980,46 @@ export class MediaManager
 			}
 		}, true);
 	 }
-	
+
+	/**
+	 * Called by User
+	 * to switch between front and back camera on mobile devices
+	 *
+	 * @param {*} streamId Id of the stream to be changed.
+	 * @param {*} facingMode it can be "user" or "environment"
+	 *
+	 * This method is used to switch front and back camera.
+	 */
+	switchVideoCameraFacingMode(streamId, facingMode)
```

Comment:
This change just provide a simple interface. User can make toggle command if s/he wants